src/solc_0.8/raffle/GenericRaffle.sol

```diff
@@ -65,6 +81,7 @@ contract GenericRaffle is ERC721EnumerableUpgradeable, OwnableUpgradeable, Reent
         address _contractAddress,
         uint256 _erc1155Id
     ) external onlyOwner {
+        require(_waveMaxTokens <= maxSupply, "_waveMaxTokens should not exceed maxSupply");
         require(_waveType < 3 && _waveMaxTokens > 0 && _waveMaxTokensToBuy > 0, "Invalid configuration");
         if (_waveType != 0) {
             require(_contractAddress != address(0x0), "Invalid contract address");
```

Comment:
I can't see the change for this comment
```_contractAddress can be an externally owner account.```

Reply:
`_contractAddress` is always `0x0`, so cannot check if it's a contract.